### DISCUSSION TITLE
Ability to delete/manipulate individual skins

### DISCUSSION THREAD

**Discussion by Sublimelime** [ppy/osu#16145] [Ideas] [answered]
As it is now, as far as I can tell, it's impossible to manipulate individual skins in the skin selector - once a skin is imported, you're stuck with it unless you delete all skins via the button at the bottom of settings (and if you want to preserve some skins, you have to export them first). You also can't do things like duplicate skins (to have a copy to edit, preserving the original).

It would be nice to have a "manage skins" UI like collections have.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/5820
